### PR TITLE
Add subscribe button to welcome window

### PR DESCRIPTION
**Pull request by 0x6431346e**
## What changed?
Added subscribe button to the welcome window

### CODE REVIEW

File: app/windows/Welcome/renderer.jsx [hunk 1]
```diff
@@ -27,12 +27,16 @@ class WelcomeWindow extends React.Component {
     window.close()
   }
 
+  handleSubscribe = () => {
```
Comment (koalalorenzo) [inline app/windows/Welcome/renderer.jsx]: as this is part of the `WelcomePage` logic we should probably move it there.
I think you can have a simple function if you don't want to rewrite it as a class

**Comment by koalalorenzo** [comment]
The green button looks awesome and well place 👍

**Comment by koalalorenzo** [comment]
Have we tested this on Windows?

**Comment by 0x6431346e** [comment]
Nope, let me try

**Pull request merged**